ui/components/app/wallet-overview/eth-overview.test.js

```diff
@@ -323,22 +323,28 @@ describe('EthOverview', () => {
       };
       const mockedStore = configureMockStore([thunk])(mockedFantomStore);
 
-      const { queryByTestId, queryByText } = renderWithProvider(
+      const { queryByTestId } = renderWithProvider(
         <EthOverview />,
         mockedStore,
       );
       const bridgeButton = queryByTestId(ETH_OVERVIEW_BRIDGE);
       expect(bridgeButton).toBeInTheDocument();
       expect(bridgeButton).toBeDisabled();
-      expect(queryByText('Bridge').parentElement).toHaveAttribute(
+      expect(bridgeButton.parentElement).toHaveAttribute(
         'data-original-title',
         'Unavailable on this network',
       );
     });
 
+    it('should always show the Receive button', () => {
+      const { queryByTestId } = renderWithProvider(<EthOverview />, store);
+      const receiveButton = queryByTestId(ETH_OVERVIEW_RECEIVE);
+      expect(receiveButton).toBeInTheDocument();
+    });
```

Comment:
Below test was mixing receive test id and looking for the portfolio button so fixing below test to look for portfolio link test id and adding new test for receive button